test/e2e/config/vsphere-ci.yaml

```diff
@@ -6,165 +6,106 @@
-  - name: cluster-api
-    type: CoreProvider
-    versions:
-    - name: v0.3.25 # latest published release in the v1alpha3 series; this is used for v1alpha3 --> v1beta1 clusterctl upgrades test only.
-      value: "http://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.25/core-components.yaml"
-      type: "url"
-      contract: v1alpha3
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-        - sourcePath: "../data/shared/metadata.yaml"
-    - name: v0.4.7 # latest published release in the v1alpha4 series; this is used for v1alpha4 --> v1beta1 clusterctl upgrades test only.
-      value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.7/core-components.yaml"
-      type: "url"
-      contract: v1alpha4
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-        - sourcePath: "../data/shared/metadata.yaml"
-    - name: v1.0.1 # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-      value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.1/core-components.yaml"
-      type: "url"
-      contract: v1beta1
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-        - sourcePath: "../data/shared/metadata.yaml"
 
-  - name: kubeadm
-    type: BootstrapProvider
-    versions:
-    - name: v0.3.25 # latest published release in the v1alpha3 series; this is used for v1alpha3 --> v1beta1 clusterctl upgrades test only.
-      value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.25/bootstrap-components.yaml"
-      type: "url"
-      contract: v1alpha3
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-        - sourcePath: "../data/shared/metadata.yaml"
-    - name: v0.4.7 # latest published release in the v1alpha4 series; this is used for v1alpha4 --> v1beta1 clusterctl upgrades test only.
-      value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.4.7/bootstrap-components.yaml"
-      type: "url"
-      contract: v1alpha4
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-        - sourcePath: "../data/shared/metadata.yaml"
-    - name: v1.0.1 # latest published release in the v1beta1 series; this is used for v1beta1 --> main clusterctl upgrades test only.
-      value: "https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.0.1/bootstrap-components.yaml"
-      type: "url"
-      contract: v1beta1
-      replacements:
-        - old: "imagePullPolicy: Always"
-          new: "imagePullPolicy: IfNotPresent"
-      files:
-      - sourcePath: "../data/shared/metadata.yaml"
+- name: cluster-api
```

Comment:
Just one minor question @DanielXiao :

Will it have any side effects if we only specify one version for v1beta1, is it safe to remove the v1alpha3 and v1alpha4 support?

Reply:
You are only installing one version of Cluster API which is v1.6.1. Dropping v1.5, v1.4, ... doesn't change anything.

The apiVersions supported will be the one supported by Cluster API v1.6.1

Reply:
Gotcha, thanks @sbueringer !